src/components/verification-grid/verification-grid.ts

```diff
@@ -71,15 +74,18 @@ interface HighlightSelection {
  * @slot - A template element that will be used to create each grid tile
  * @slot - Decision elements that will be used to create the decision buttons
  * @slot data-source - An `oe-data-source` element that provides the data
+ * @slot classifications - Grouped `oe-classification` components
+ * @slot verifications - Grouped `oe-verification` components
  *
  * @fires decision - Emits information about the decision that was made
  * @fires loaded - Emits when all the spectrograms have been loaded
  */
 @customElement("oe-verification-grid")
 export class VerificationGridComponent extends AbstractComponent(LitElement) {
   public static styles = unsafeCSS(verificationGridStyles);
-  private static loadedEventName = "loaded" as const;
-  private static decisionMadeEventName = "decision-made" as const;
+
+  public static readonly decisionMadeEventName = "decision-made" as const;
+  private static readonly loadedEventName = "loaded" as const;
```

Comment:
maybe future issue: our events should be namespaced

Reply:
Related issue: https://github.com/ecoacoustics/web-components/issues/142